src/utilities/theme/role-variants.ts

```diff
@@ -240,6 +240,12 @@ export const roleVariants: RoleVariants = {
       light: {lightness: 82},
       dark: {lightness: 18},
     },
+    {
+      name: 'iconOnInteractive',
+      description: 'For use as a fill color for icons on critical actions.',
+      light: {lightness: 100},
+      dark: {lightness: 100},
```

Comment:
Seems like one of these values should be adjusted. Usually their values should be close to eachothers inverse color. i.e. if light has a lightness of 80, then dark would have a lightness of 20.

Reply:
In this case, the icon is white in both cases. Do we have other cases like this?